test/integration/forms/form.js

```diff
@@ -1318,11 +1305,31 @@ context('Patient Action Form', function() {
         expect(storage.saveButtonType).to.equal('saveAndGoBack');
       });
 
+    cy
+      .iframe()
+      .find('textarea[name="data[familyHistory]"]')
+      .type('Here is some typing');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[storyTime]"]')
+      .type('Once upon a time...');
+
     cy
       .get('.form__controls')
       .find('.js-save-button')
       .click();
 
+    cy
+      .get('.form__controls')
+      .find('.js-save-button')
+      .should('be.disabled');
+
+    cy
+      .get('.form__controls')
+      .find('.button__drop-list-select')
+      .should('be.disabled');
```

Comment:
Tests that both the save button and droplist button are `disabled` after a user saves the form.

Do we need to add more delay on the stubbed `@routePostResponse` route? It's currently set to `delay: 100` earlier on in this test. Or is `100ms` enough time for Cypress to grab those buttons before the page is redirected?

Reply:
well the communication between the iframe and the app is `postMessage` which is async.. so in this case the delay time is an unknown, but surely 100ms is enough.

Reply:
ah, but I suppose it could be faster than that..